packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -89,10 +111,23 @@ Future<void> main(List<String> args) async {
 
   loadPathOpsIfNeeded(results);
 
-  final String inputFilePath = results['input'] as String;
-  final String xml = File(inputFilePath).readAsStringSync();
-  final File outputFile =
-      File(results['output'] as String? ?? '$inputFilePath.vg');
+  final List<Pair> pairs = <Pair>[];
+  if (results.wasParsed('input-dir')) {
+    final Directory directory = Directory(results['input-dir'] as String);
+    for (final File file
+        in directory.listSync(recursive: true).whereType<File>()) {
+      if (!file.path.endsWith('.svg')) {
+        continue;
+      }
+      final String outputPath = '${file.path}.vg';
```

Comment:
I think we're using `.vec` in some places and `.vg` in others. So this works fine for the current target but will get weird for others.

Reply:
I just updated this to use .vec, might as well